operators/multiclusterobservability/controllers/placementrule/placementrule_controller.go

```diff
@@ -93,6 +96,25 @@ func (r *PlacementRuleReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	reqLogger := log.WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name)
 	reqLogger.Info("Reconciling PlacementRule")
 
+	// ACM 8509: Special case for hub/local cluster metrics collection
+	// We want to ensure that the local-cluster is always in the managedClusterList
+	// In the case when hubSelfManagement is enabled, we will delete it from the list and modify the object
+	// to cater to the use case of deploying in open-cluster-management-observability namespace
+	delete(managedClusterList, "local-cluster")
```

Comment:
in future, we should totally remove the dependencies of `local-cluster` and `obsaddon` for this case.